cli/command/migrate.go

```diff
@@ -152,7 +152,7 @@ func checkMigrateTopology(curveadm *cli.CurveAdm, data string) error {
 		dcs2add[0].GetRole() != dcs2del[0].GetRole() {
 		return errno.ERR_REQUIRE_SAME_ROLE_SERVICES_FOR_MIGRATING
 	}
-	if len(dcs2del) != dcs2del[0].GetReplicas() {
+	if len(dcs2del) != dcs2del[0].GetInstance() {
```

Comment:
Sorry, i didn't express clearly, actually we should use word instance**s** for multi-instances.